src/vs/workbench/contrib/debug/browser/callStackEditorContribution.ts

```diff
@@ -29,6 +29,8 @@ const stickiness = TrackedRangeStickiness.NeverGrowsWhenTypingAtEdges;
 const TOP_STACK_FRAME_MARGIN: IModelDecorationOptions = {
 	description: 'top-stack-frame-margin',
 	glyphMarginClassName: ThemeIcon.asClassName(debugStackframe),
+	glyphMargin: { position: GlyphMarginLane.Right },
+	zIndex: 9999,
```

Comment:
The glyph margin and editor don't know anything about debug decorations vs non-debug decorations--all they know about is the specified `zIndex` and `position`. To ensure that the debug frame indicator continues to render together with the breakpoint decoration, we need both options to match the breakpoint decorations options, which is where these values come from.